ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java

```diff
@@ -116,10 +124,23 @@
   public static void main(String args[]) throws Exception {
     OptionParser parser = new OptionParser();
 
+    ArgumentAcceptingOptionSpec<String> typeOfOperationOpt =
+        parser.accepts("typeOfOperation", "Type of operation. Supported operations: UpdateEntireJson, EnableEncryption")
+            .withRequiredArg()
+            .describedAs("typeOfOperation")
+            .ofType(String.class);
+
+    ArgumentAcceptingOptionSpec<String> enableEncryptionForOpt = parser.accepts("updateEncryptionFor",
```

Comment:
not sure why we need a different option for this - if you update the account json in the file path correctly, should be enough? This is what the tool seems to do for any other change so I'm not sure why encryption should be treated any differently. Also introduces unnecessary complexity in terms of arguments/input etc. Any problems doing it that way?

Reply:
I want to simplify the account update tool going forward. Requiring to send in the entire json as input is not trivial and error prone. We have to fetch the account json from helix, edit it manually and then send it as input to the tool. But from now on, we might have lot of use-cases where in we need to update some specific properties of an account or container like encryption field, media scan enabled, cache related settings and so on. Just hold on for some time. I will try to make this argument generic which could be extended further by just introducing new type of operation. For eg: UpdateMediaScan, and so on. The argument "enableEncryptionForOpt" (will have to be renamed) will take in accountContainerPropertyValuePair( in this context, account1:container1:true,account2:container1:false). Depending on the operation type, the value will be interpreted. In this case, it refers to media scan. Will let you know once the patch is updated.

Reply:
Sounds good - however before you make any changes, lets have a chat with @cgtz.